### ISSUE TITLE
[0.49.19] RFE - Show Cluster Roll As Hyperlink in Combat Log

### ISSUE THREAD

**Issue by PhoenixHeart512** [MegaMek/megamek#5437]
### Environment
megamek.MegaMek.initializeLogging(MegaMek.java:118) - Starting MegaMek v0.49.19
	Build Date: 2024-04-19T14:55:55.055536
	Today: 2024-04-30
	Origin Project: MegaMekLab
	Java Vendor: Eclipse Adoptium
	Java Version: 17.0.6
	Platform: Windows 10 10.0 (amd64)
	System Locale: en_US
	Total memory available to MegaMek: 8 GB

### Description
As it says on the tin, I would like to have the cluster roll result show the roll result for the cluster roll as a hyperlink so that the player can click to see what the 2d6 result was. While I'm 99% sure it's just player bias, it feels like there are an abnormally large number of maximum cluster results that happen in MegaMek, and being able to see the dice result would go a long way towards dispelling that superstition.

It would also help to have the cluster modifier hyperlinked so that the player can see what modifiers affected the roll, similar to what we already do for the to-hit TN ("needs 8").

In the screenshot below, the roll result pop-up is from the To-Hit roll result as an example of what I would LIKE to see for the cluster roll (boxed in red)

![image](https://github.com/MegaMek/megamek/assets/101755917/7ecc8375-3048-448a-a83e-33c2fb0a836d)

**Comment by PhoenixHeart512** [comment]
True, but my personal bias is a concern that it's not actually rolling the dice like it says it is lol.

Plus I'm always a fan of more info = better, especially if it's only presented when you click on a thing to view it.

**Comment by RaozSpaz** [comment]
Could this be added as a sort of GM option that way players who want to see it or are testing things can see it but its not always showing by default.

**Comment by celestiun** [comment]
as a huge fan of cluster weapons. 
yes please?

**Comment by cooshoo** [comment]
Would also help debugging effects affecting cluster table rolls.
